src/App.tsx

```diff
@@ -23,6 +28,13 @@ const App = () => {
     initDerivAnalytics();
     redirectToOauth();
 
+    useEffect(() => {
+        if (isGBLoaded) {
+            if (!isP2PV2Enabled) {
+                window.location.href = 'https://app.deriv.com/cashier/p2p';
```

Comment:
I don't think we should hardcode this in the code? how about Utils?

Reply:
You are right, Let me see where can i put it.